pytorch_to_returnn/torch/nn/modules/conv.py

```diff
@@ -78,14 +78,24 @@ def create_returnn_layer_dict(self, input: Tensor) -> Dict[str, Any]:
     assert all(p == 0 for p in self.padding)  # not implemented otherwise
     assert all(p == 0 for p in self.output_padding)  # not implemented otherwise
     assert self.padding_mode == "zeros"  # not implemented otherwise
+
+    from pytorch_to_returnn.naming import Naming
+    naming = Naming.get_instance()
+    input_tensor = naming.tensors[input]
+    in_dim = input_tensor.returnn_data.dim_tags[input_tensor.returnn_axis_from_torch_axis[1]]
+    in_spatial_dims = [
+      input_tensor.returnn_data.dim_tags[input_tensor.returnn_axis_from_torch_axis[dim + len(input.shape)]]
```

Comment:
Because that would return an axis description like `"T"` or `"F"`. This would be mapped to a dim tag in the `ConvLayer` construction. However, in case we do convolution over the feature dim, `"F"` would be mapped to the `in_dim`, so the new feature dim and not the old one which does not work.